common/info.yaml

```
@@ -297,6 +313,295 @@ description: |
   If you need to implement a retry mechanism in your integration,
   please contact the Bond Engineering Team via the Bond Forums https://forum.bondhome.io/
 
+
+  ## Modified Times
+
+  Certain endpoints may track their last modified time in a special
+  `__modified` field. The modified time is an integer representing
+  the time in seconds since the Unix epoch when the endpoint was last modified.
+
+  Currently the endpoints exposing `__modified` are:
+
+      groups/{}
+      groups/{}/skeds
+      groups/{}/skeds/{}
+      scenes/{}
+      scenes/{}/skeds
+      scenes/{}/skeds/{}
+
+  # Groups and Scenes
+
+  The Group, Scene, and related Skeds endpoints are special in that they
+  represent shards of abstract objects possibly distributed across
+  multiple Bond Bridges or Smart by Bond devices. We will here discuss:
+
+   - discovery, control, and feedback (for all API clients)
+   - management (for Bond Home app)
+
+  The following discussion is in terms of Groups but should be understood
+  as also applying to Scenes unless otherwise specified.
+
+  ## Discovery
+
+  To discover the Groups available on a Bond network,
+  the client should first query the `groups/` enumeration
+  to get the group IDs in which each Bond participates.
+  The union of the Group IDs gives the list of Groups on the network.
+
+  For example:
+
+      ZZBL12345:groups = {
+        "0000000000000001":{...},
+        "0000000000000002":{...},
+        "0000000000000003":{...}
+      }
+
+      KSMJWCE12345:groups = {
+        "0000000000000002":{...},
+        "0000000000000004":{...}
+      }
+
+  There are a total of 4 groups on the network:
+
+      "0000000000000001"
+      "0000000000000002"
+      "0000000000000003"
+      "0000000000000004"
+
+  The Bond Bridge (ZZ...) participates in three groups with IDs 1, 2, and 3,
+  while a Smart by Bond fan (K...) participates in groups 2 and 4.
+
+  Next, the client should query each of the group shard endpoints `group/{}`.
+  The list of devices participating in the group is the union of the devices
+  in the group shards.
+
+  For example, if the result of querying the `groups/{}` endpoint for the
+  group with ID 1 is:
+
+      ZZBL12345:groups/0000000000000001 = {
+        "__modified": 1642788966,
+        "name": "Great Room Fans",
+        "types": ["CF"],
+        "locations": ["Great Room"],
+        "devices": ["aabbccdd", "11223344"],
+        "actions": ["TurnOn", "TurnOff"]
+      }
+
+      KSMJWCE12345:groups/0000000000000001 = {
+        "__modified": 1642788968,
+        "name": "Great Room Fans",
+        "types": ["CF"],
+        "locations": ["Great Room"],
+        "devices": ["1"]
+        "actions": ["TurnOn", "TurnOff", "TurnLightOn", "TurnLightOff"]
+      }
```

I believe you meant ID 2. ID 1 is ZZ only.
```suggestion
  group with ID 2 is:

      ZZBL12345:groups/0000000000000002 = {
        "__modified": 1642788966,
        "name": "Great Room Fans",
        "types": ["CF"],
        "locations": ["Great Room"],
        "devices": ["aabbccdd", "11223344"],
        "actions": ["TurnOn", "TurnOff"]
      }

      KSMJWCE12345:groups/0000000000000002 = {
        "__modified": 1642788968,
        "name": "Great Room Fans",
        "types": ["CF"],
        "locations": ["Great Room"],
        "devices": ["1"]
        "actions": ["TurnOn", "TurnOff", "TurnLightOn", "TurnLightOff"]
      }
```